### ISSUE TITLE
TimeSeriesDataset restricted to four decimals? 

### ISSUE THREAD

**Issue by flxshk** [jdb78/pytorch-forecasting#872]
- PyTorch-Forecasting version: 0.9.2
- PyTorch version: 1.10.1+cu102
- Python version: 3.8
- Operating System:Debian 4.19.208-1

First off, thank you for this great project. 

### Behavior

Analyzing coordinates with reasonable precision requires 6 decimals in my case. When I cast my data as a TimeSeriesDataSet, only four decimals are considered. 

Is there a programmatic way to change this behavior, i.e., to consider all 6 decimals when creating the TimeSeriesDataSet and predicting with 6 decimals? 

 
Edit: 
- TimeSeriesDataSet Normalizers/Encoders have a numerical stability parameter: 
- E.g.,:  eps (float, optional) – Number for numerical stability of calculations. Defaults to 1e-8.

Yet, visualizing the data shows 4 decimals and also the predictions have 4 decimals. Also, I did not specify that the displayed output should be restricted to 4 decimals, thus I expect it is the behavior of the TimeSeriesDataSet module to either calculate with 4 decimals or to display only 4 decimals.

**Comment by flxshk** [comment]
This is the output from a dataloader batch. Encoder input has more than 4 decimals, but encoder target has 4 decimals. 

`'encoder_cont': tensor([[[ 1.0000e+00, -1.6215e+00, -1.0000e+00,  ...,  4.7783e-01,
            6.4915e-01,  1.1946e-01], ....

 'encoder_target': [tensor([[0.8396, 0.8392, 0.8389,  ..., 0.8384, 0.8384, 0.8384], ....

`
